.travis.yml

```diff
@@ -0,0 +1,5 @@
+script:
+- docker build --tag travis_todo:latest --target test .
+- docker run travis_todo todo_app/test/test_unit.py
+- docker run --env-file .env.test travis_todo todo_app/test/test_appTest.py
+- docker run travis_todo todo_app/test/test_appUiTest.py
```

Comment:
Here you're not passing any environment variables through, so how will this test be able to communicate with Trello?

Reply:
Jack I have acted on your comment. Please review again

Reply:
Thanks, feel free to delete this file now we've switched to Github Actions

Reply:
deleted